src/Elements/Geometry/Containment.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+namespace Elements.Geometry
+{
+    /// <summary>
+    /// Represents the state of containment of a point relative to an enclosing polygon, profile, or volume
```

Comment:
`The containment of a point relative to an enclosing polygon, profile, or solid.`